experimental/FTheoryTools/src/auxiliary.jl

```diff
@@ -176,24 +176,45 @@ function _kodaira_type(id::MPolyIdeal{T}, f::T, g::T, d::T, ords::Tuple{Int64, I
     poly_g = ring_map(g)
     poly_d = ring_map(d)
     locus = ring_map(gens(id)[1])
+    locus_id = ideal([locus])
     
     if f_ord == 0 && g_ord == 0
-      monodromy_poly = _psi^2 + divexact(evaluate(9 * poly_g, [locus], [0]), evaluate(2 * poly_f, [locus], [0]))
+      quotient_gens = gens(groebner_basis(quotient(ideal([9 * poly_g]), ideal([2 * poly_f])) + locus_id))
+      deleteat!(quotient_gens, findall(g -> g == locus, quotient_gens))
+      quotient_val = quotient_gens[1]
```

Comment:
I am wondering if the check `g == locus` will always work. Generators of the quotient are not unique. Will the Groebner basis computation completely eliminate this redundancy? Also, could additional signs/numeric factors appear that would force `g == locus` to return  `false`?

Less critical, but relevant: Unless I am mistaken this pattern of lines appears four times in this PR. Does it make sense to add an auxiliary function and call it instead to make the code shorter/cleaner?